packages/react-components/src/components/FileCard.tsx

```diff
@@ -98,7 +99,12 @@ export const _FileCard = (props: _FileCardProps): JSX.Element => {
   };
 
   return (
-    <Stack className={containerClassName}>
+    <Stack
+      className={containerClassName}
+      onClick={() => {
```

Comment:
Is this focusable by keyboard nav/screen reader? What does the screen reader read out when focused?

Reply:
Good catch! Turns out the whole file card is not tab accessible, let me look into a solution to this or should this be a separate bug?

Reply:
Probably best as a seperate bug and PR as I imagine it will add new strings etc and sometimes a11y fixes turn out to be quite large

Reply:
Yeah, this is being fixed. I have PR up for the accessibility issue

Reply:
@carocao-msft I think we should wait for the PR(https://github.com/Azure/communication-ui-library/pull/1881) to be merged. It is using icon buttons which might resolve the issue. My concern is if we make the outer stack clickable and the file card is not tab navigable. User might not be able to trigger onclick using keyboard.

Reply:
Merging this PR for now. Will resolve accessibility issue if it occurs once https://github.com/Azure/communication-ui-library/pull/1881 is merged in